.travis.yml

```diff
@@ -1,16 +1,15 @@
+sudo: false
 language: python
 env:
-  - REQUESTS_VERSION=2.1.0
-  - REQUESTS_VERSION=2.8.1
-  - REQUESTS_VERSION=2.9.1
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-install:
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.6* ]]; then export PYTEST_ADDOPTS='-p no:pylint'; fi
-  - pip install -r requirements.txt
-  - pip install --upgrade --force-reinstall requests==$REQUESTS_VERSION
-script: py.test
+    - TOXENV=py27-requests-current
```

Comment:
You could run tox without any TOXENV vars but that would collapse all the environments into a single one. Tox would then run all permutations but it would be harder to which tests failed from the travis interface.

By defining the different environments it forces travis to display the results for the different environments.